support-frontend/assets/pages/digital-subscription-checkout/thankYouExisting.jsx

```diff
@@ -38,10 +38,7 @@ const content = (
     <Page
       header={<Header countryGroupId={countryGroupId} />}
       footer={
-        <Footer
-          faqsLink="https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions"
```

Comment:
I thought the idea was to send the faqs to the help centre, not to delete them completely?

Reply:
true I didn't quite read the email properly, maybe we should hold off merging until someone? customer operations? has said they are happy no content is lost?

Reply:
Yes, I think we still need a link in all these places, just to the help centre rather than the faqs